include/param.h

```diff
@@ -225,8 +232,6 @@ class Params
 
 public:
   static constexpr uint8_t PARAMS_NAME_LENGTH = 16;
-
-private:
   union param_value_t
```

Comment:
why did this need to become public?

Reply:
Did I make this public? Not a big deal, but I tend to make constants public. Especially here, because the parameter names pass through other classes, such as CommLink.

Reply:
Yeah, but they don't need to see this union; that's an internal implementation thing. I'd revert this to private

Reply:
OK, I see now that I'm looking at the right page on Github.

I needed to make the `params_t` struct public, so that the memory manager could hold one. Strangely, `param_value_t` does not need to be public for that to work, even though it is a member of `params_t`